structured/structured.proto

```diff
@@ -84,3 +84,16 @@ message TableDescriptor {
   optional uint32 next_index_id = 6 [(gogoproto.nullable) = false,
       (gogoproto.customname) = "NextIndexID"];
 }
+
+// DatabaseDescriptor represents a namespace (aka database) and is stored
+// in a structured metadata key. The DatabaseDescriptor has a globally-unique
+// ID shared with the TableDescriptor ID.
+// Permissions are applied to all tables in the namespace.
```

Comment:
On Thu, Jul 30, 2015 at 10:19 AM mberhault <notifications@github.com> wrote:

> In structured/structured.proto
> <https://github.com/cockroachdb/cockroach/pull/1857#discussion_r35874625>:
>
> > @@ -84,3 +84,16 @@ message TableDescriptor {
> >    optional uint32 next_index_id = 6 [(gogoproto.nullable) = false,
> >        (gogoproto.customname) = "NextIndexID"];
> >  }
> > +
> > +// DatabaseDescriptor represents a namespace (aka database) and is stored
> > +// in a structured metadata key. The DatabaseDescriptor has a globally-unique
> > +// ID shared with the TableDescriptor ID.
> > +// Permissions are applied to all tables in the namespace.
>
> nope. For now tables do not have permissions. The next step is to apply
> permissions at the database level. After that comes per-table permissions
> that are inherited from the database-level permissions.
>
Sounds fine. Although, it's the same as defaults that cannot be overridden
at this stage.

Please check it in so that I can rebase against it. Thanks


> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1857/files#r35874625>.
>